pentaho-karaf-assembly/src/main/filtered-resources/etc-server/custom.properties

```diff
@@ -94,8 +94,8 @@ org.osgi.framework.system.packages.extra= \
  org.pentaho.vfs.ui, \
  org.pentaho.xul.swt.tab, \
  org.slf4j.*; version\="1.7.7", \
- org.springframework.dao; version\="4.3.2.RELEASE", \
- org.springframework.security.*; version\="4.1.5.RELEASE", \
+ org.springframework.dao; version\="4.3.19.RELEASE", \
```

Comment:
Please use maven filtering to fill in the version value.

Reply:
Agree, working on it